packages/ui/package.json

```diff
@@ -116,6 +116,7 @@
     "antd": "^4.24.3",
     "classnames": "^2.3.2",
     "react-markdown": "^6",
+    "remark-directive": "^1.0.1",
```

Comment:
It doesn't looks like a maintainable package! 
It less then 50 LOC. And less then 1k stars. 
Do we really need it?

Reply:
We use the package [react-markdown](https://www.npmjs.com/package/react-markdown) to convert markdown to react elements. To add custom syntax to markdown for elements such as tables or alerts, react-markdown suggests using plugins written for it. react-markdown mentions in its documentation a [list of plugins](https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins) that can be used with it.

We already use the [remark-gfm](https://www.npmjs.com/package/remark-gfm) plugin for custom table syntax which also has less than 1k stars. And [remark-directive](https://www.npmjs.com/package/remark-directive) is the most popular among all plugins which fit the right functionality for the task and which the react-markdown documentation refers to.